StartupReasonReporter/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
```

Comment:
Please revert this change.

Reply:
Sure, you don't want the version number to match with the released carthage version?

Reply:
Yeah, the version numbers mismatch, but we'll fix it when we release. Since it's already broken this is fine.